extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -97,7 +97,11 @@ public void handle(RoutingContext event) {
                 response.headers().set(HttpHeaders.ACCESS_CONTROL_ALLOW_ORIGIN, origin);
             }
 
-            response.headers().set(HttpHeaders.ACCESS_CONTROL_ALLOW_CREDENTIALS, "true");
+            boolean allowCredentials = corsConfig.accessControlAllowCredentials
+                    || (corsConfig.origins.isPresent() && !corsConfig.origins.get().contains("*")
```

Comment:
@mcserra Looks like we can drop `!corsConfig.origins.get().contains("*")` because we have `corsConfig.origins.get().contains(origin)` which ensures the precise match, but instead we should check that the `Origin` header does no contain `*`.
This will ensure that, assuming `Origin` is something like `https://origin`, then if we have the allowed origins as `https://origin,*` then `true` will be returned due to a precise match.
Do you agree ?

Reply:
I do. I'll do a commit.